src/tap/src/tapcell.tcl

```diff
@@ -191,11 +191,13 @@ proc tapcell { args } {
     }
   }
 
+
+  if { [info exists keys(-tapcell_master)] } {
   tap::clear
```

Comment:
I appears to me that adding this just bypasses the `tapcell` command all together. It seems to me that the proper solution is to not call `tapcell`, if you don't want tapcells inserted.

Reply:
@arlpetergadfort You're right. @shuolai  To skip the `tapcell` insertion using the flow, you can create a custom `tapcell.tcl` file for your design that will not call the `tapcell` command.

Reply:
@arlpetergadfort @eder-matheus Thank you for your advice. My issue is that I would like to `insertEndcaps` and skip `insertTapcells` at the same time. So I think I have to call `tapcell` function. I tried to remove `tapcell_master` in `tapcell.tcl` but I got the following error message.

Maybe adding a if statement to skip line 156 and 157 in tapcell.cpp could be a solution to this problem? 

![tapcell_tcl](https://user-images.githubusercontent.com/79955802/132398165-2ff8b5ba-d749-42ae-ae81-3e623aeb9110.png)
![error](https://user-images.githubusercontent.com/79955802/132398174-4b4e341b-af44-4024-82f7-b60e66892ebd.png)